Utilities/DataCompression/include/DataCompression/TruncatedPrecisionConverter.h

```diff
@@ -18,32 +18,29 @@
 //  @author Matthias Richter
 //  @since  2015-08-08
 //  @brief  A simple converter producing truncated precision
-//          according to a data model 
-
-namespace AliceO2 {
-
-template<class _ParameterModel>
-class TruncatedPrecisionConverter {
-public:
+//          according to a data model
+
+namespace AliceO2
+{
+template <class _ParameterModel>
+class TruncatedPrecisionConverter
+{
+ public:
```

Comment:
Doesn't:

```
AccessModifierOffset 2
```

do what we want? We currently have "-1", at least on the web page.